bin/promote-nightly-to-release

```diff
@@ -211,6 +211,10 @@ function statusline() {
   tput sgr0
 }
 
+statusline "Reading git config"
```

Comment:
add a code comment explaining that this intentionally reads the configuration from packaging/.git/config if present, and it falls back to the global config if not